tests/pactHelper.js

```diff
@@ -342,7 +342,7 @@ const getUserInformationAsAdminInteraction = function (provider) {
 }
 
 const getCapabilitiesWithInvalidAuthInteraction = function (provider, username = config.adminUsername, password = config.invalidPassword) {
-  return provider.uponReceiving(`a capabilities GET request with invalid authentication by ${username} with ${password}`)
+  return provider.uponReceiving(`a GET request to get capabilities with invalid auth by '${username}' with '${password}'`)
```

Comment:
should be `as user ..` if we follow the same structure like in the other requests

Reply:
the decision was made to use `as <username>, a <METHOD> request ...`